cirq/contrib/two_qubit_gates/gate_compilation_test.py

```diff
@@ -0,0 +1,59 @@
+"""Tests for gate_compilation.py"""
+import numpy
+import pytest
+
+# pylint: disable=line-too-long
+from cirq import unitary, FSimGate
+from cirq.contrib.two_qubit_gates.gate_compilation import gate_product_tabulation
+from cirq.contrib.two_qubit_gates.math_utils import random_two_qubit_unitaries_and_kak_vecs, unitary_entanglement_fidelity
+
+# pylint: enable-line-too-long
+
+# coverage: ignore
+numpy.random.seed(11)  # for determinism
```

Comment:
Do not affect global state during tests. Especially not at module import time. You have no guarantee that you are not affecting other tests by doing this, nor that other tests will not generate random numbers and break the tests here.

Instead, you should modify the tabulation method to take a `random_state: Optional[Union[np.random.RandomState, int]] = None`. Copy what `cirq.testing.random_circuit` does.

Reply:
It's also done in fidelity_estimation_test.py and clifford_simulator_test.py

Reply:
Done.